From 036a47935ff3852f9d3b0feb8f4967d24799a5de Mon Sep 17 00:00:00 2001
From: Sammy He <r62914@freescale.com>
Date: Wed, 14 Dec 2011 23:35:24 +0800
Subject: [PATCH 252/505] ENGR00170228 vpu: Add printing detail f/w source
 code version for mx6

The f/w source code version report has been added into vpu_GetVersion()
function in vpu lib in this CQ change, change vpu unit test to print
the detail f/w source code version here.

Signed-off-by: Sammy He <r62914@freescale.com>
---
 test/mxc_vpu_test/main.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/test/mxc_vpu_test/main.c b/test/mxc_vpu_test/main.c
index 7c4c60a..32daa8e 100644
--- a/test/mxc_vpu_test/main.c
+++ b/test/mxc_vpu_test/main.c
@@ -412,8 +412,8 @@ main(int argc, char *argv[])
 		return -1;
 	}
 
-	info_msg("VPU firmware version: %d.%d.%d\n", ver.fw_major, ver.fw_minor,
-						ver.fw_release);
+	info_msg("VPU firmware version: %d.%d.%d_r%d\n", ver.fw_major, ver.fw_minor,
+						ver.fw_release, ver.fw_code);
 	info_msg("VPU library version: %d.%d.%d\n", ver.lib_major, ver.lib_minor,
 						ver.lib_release);
 
-- 
1.8.0

